exchange/howto/manual-dispute-payout.adoc

```diff
@@ -0,0 +1,184 @@
+
+sellerPrivateKeyAsHex:: The seller's private key (leave empty if seller was not the winner)
+
+arbitratorPrivateKeyAsHex:: The arbitrator's private key
+
+buyerPubKeyAsHex:: The buyer's public key (leave empty if buyer was not the winner)
+
+sellerPubKeyAsHex:: The seller's public key (leave empty if buyer was not the winner)
+
+arbitratorPubKeyAsHex:: The arbitrator's public key
+
+P2SHMultiSigOutputScript:: The P2SH multi-sig output script hex value as copied from the block explorer
+
+When all values have been entered (and double-checked!), click `Sign and publish transaction` to issue the payout.
+
+When you have confirmed that the payout transaction worked as expected, *let the trader know and close out the ticket*.
+
+
+== Trader instructions
+
+[WARNING]
+The instructions below require you to share sensitive private key information with your arbitrator in order to assist in a manual multi-sig payout. This is necessary only in extraordinary situations, e.g. when a bug or network issue has caused the normal, automated multi-sig payout process to fail. In the unlikely event that you do need to follow these instructions, *you should never share private key information with anyone other than your arbitrator*, and *you should send your private key information only via the Bisq support ticket chat interface*.
+
+=== Step 1. Get your public key from your arbitrator
+
+Your arbitrator will follow the link:#arbitrator-instructions[instructions] above to determine which of your public keys was used to participate in the multi-sig transaction.
+
+Wait for your arbitrator to provide you with this value. They will send it to you via the Bisq support ticket chat interface. When you have received the public key from the arbitrator, move on to the next step.
+
+=== Step 2. Remove your wallet password [[remove-password]]
+
+Removing your wallet password is necessary in order to get the unencrypted value of your corresponding private key in the next step.
+
+Go to `Account > Wallet Password`, enter your password and click the `Remove password` button:
+
+image:images/password-active.png[wallet password active]
+
+When your wallet password has been removed, you should see the following:
+
+image:images/password-removed.png[wallet password removed]
+
+You can now move on to the next step.
+
+=== Step 3: Send your private key to your arbitrator
+
+Press `cmd+j` on macOS or `ctrl-j` on Windows to *open Bisq's Wallet Data dialog*. You should see a screen similar to the following:
+
+image:images/wallet-data.png[wallet data dialog]
+
+At the bottom of the dialog, *check the box that reads `Include private keys`*:
+
+image:images/include-private-keys.png[include private keys]
+
+Then *click `Copy to clipboard`*.
+
+Open the text editor of your choice, e.g. TextEdit on macOS or Notepad on Windows, and *paste the copied wallet data text* into it.
+
+In the text editor, *search for the public key value your arbitrator provided* in step one above.
```

Comment:
Be sure the found text gets followed by "DeterministicKey{pub HEX=....". There are also all transactions listed in the text and you might find the public key there as well. Continue your search until you are at the address entry with the keys.